setup.py

```diff
@@ -76,6 +76,7 @@ def run(self):
     install_requires=[
         'pyside6>=6.5.1',
         'Pillow>=11.3.0',
+        'PyMuPDF>=1.18.0',
```

Comment:
Why this version? The latest is 1.26.x

Reply:
No specific reason, probably copied without thinking from an old reference when looking up how the library worked. I still see 1.16.0 on their own docs sometimes like [here](https://pymupdf.readthedocs.io/en/latest/tutorial.html)

Reply:
I see it's already bumped with your merge